zebra/tc_netlink.c

```diff
@@ -0,0 +1,433 @@
+/*
```

Comment:
You will likely need to make a `zebra/tc_socket.c/h`

Take a look at `zebra/rule_socket.c/h` as an example

Reply:
Basically just servers as a way to make sure it complies on BSD.